src/gui/autoexpandabledialog.cpp

```diff
@@ -47,11 +47,24 @@ QString AutoExpandableDialog::getText(QWidget *parent, const QString &title, con
                                       Qt::InputMethodHints inputMethodHints) {
 
   AutoExpandableDialog d(parent);
+// ignore .suffix when renaming
+  QString f_find = ".";
+  int f_len = text.length();
+  int f_end = text.lastIndexOf(f_find,-1);
```

Comment:
`f_len` is not used.

Anyway, you could replace these three lines with a simple
```
int f_end = text.lastIndexOf(".");
```